festung/dbapi.py

```diff
@@ -154,6 +152,8 @@ def execute(self, operation, parameters=None):
         res = self._request('POST', json=data)
         # For debugging purposes
         self._data = res
+        if 'errors' in res:
+            raise DataError(res['message'], errors=res['errors'])
```

Comment:
Put a `FIXME` because there's an error type, which is not supported by festung yet. And sometimes festung should be able to raise an `OperationalError`.